map/routing_manager.cpp

```diff
@@ -136,6 +138,8 @@ RouteMarkData DeserializeRoutePoint(json_t * node)
   FromJSONObject(node, "x", data.m_position.x);
   FromJSONObject(node, "y", data.m_position.y);
 
+  FromJSONObject(node, "replaceWithMyPosition", data.m_replaceWithMyPosition);
```

Comment:
Where is this json stored? Is there a possibility that old version stored old json, and newly updated version failed to load old json? Won't it crash?

Reply:
That should be just a `FromJSONObjectOptional`, I believe. It will write the value to there if it exists, otherwise leave it (in this case, in the `false` default).